recipes/bahamas/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "bahamas" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/bahamas-{{ version }}.tar.gz
+  sha256: a89fe4c2a5d670c9ac5042da965541de0247922daf4172ad8763d91e16e39f9f
+
+build:
+  entry_points:
+    - bahamas-inference = bahamas.utilities.run_pe:main
+    - bahamas-data = bahamas.utilities.run_data:main
+    - bahamas-input = bahamas.utilities.run_input:main
+  skip: True  # [win]
+  #noarch: python
```

Comment:
It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.